File: Freedom/微能量/新闻动态/EnergyNewsDetailViewController.m
//  EnergyNewsDetailViewController.m
//  Created by Super on 16/9/5.
//  Copyright © 2016年 Super. All rights reserved.
//
#import "EnergyNewsDetailViewController.h"
@implementation EnergyNewsDetailViewController{
    UIWebView *web;
}
-(void)viewDidLoad{
    [super viewDidLoad];
    NSURL *url = [NSURL URLWithString:@"http://weibo.com/tv/v/ErdijBWkT?ref=feedsdk&type=ug"];
    NSURLRequest *request = [NSURLRequest requestWithURL:url];
    web = [[UIWebView alloc]initWithFrame:self.view.bounds];
    [web loadRequest:request];
    [self.view addSubview:web];
}
@end

File: Freedom/微能量/新闻动态/EnergyNewsDetailViewController.swift
//
//  EnergyNewsDetailViewController.swift
//  Freedom
import UIKit
class EnergyNewsDetailViewController: EnergyBaseViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        let web = UIWebView(frame:view.bounds)
        let url = URL(string:"http://weibo.com/tv/v/ErdijBWkT?ref=feedsdk&type=ug")
        let request = URLRequest(url: url!)
        web.loadRequest(request)
        view.addSubview(web)
    }
}
